File: src/main/java/org/artembogomolova/demo/webapp/security/UserDetailsServiceImpl.java
package org.artembogomolova.demo.webapp.security;

import java.util.List;
import org.springframework.security.core.userdetails.User;
import org.springframework.security.core.userdetails.UserDetails;
import org.springframework.security.core.userdetails.UserDetailsService;
import org.springframework.stereotype.Service;

@Service
public class UserDetailsServiceImpl implements UserDetailsService {

  @Override
  public UserDetails loadUserByUsername(String username) {
    return new User(username, "password", List.of());
  }
}

File: app/src/main/kotlin/org/artembogomolova/demo/webapp/main/security/UserDetailsServiceImpl.kt
package org.artembogomolova.demo.webapp.main.security

import org.springframework.security.core.userdetails.User
import org.springframework.security.core.userdetails.UserDetails
import org.springframework.security.core.userdetails.UserDetailsService
import org.springframework.stereotype.Service

@Service
class UserDetailsServiceImpl : UserDetailsService {
    override fun loadUserByUsername(username: String): UserDetails {
        return User(username, "password", listOf())
    }
}